### PR TITLE
Allow Liquid variables within the remote_include plugin

### PR DESCRIPTION
**Pull request by nickvigilante**
Allow Liquid variables within the remote_include plugin

### CODE REVIEW

**Comment by ianjevans** [comment]
NIce! Can you bump the version to 1.1.5 in `jekyll-remote-include.gemspec`?

**Comment by nickvigilante** [comment]
It was already at 1.1.5, so I bumped it to 1.1.6.

File: lib/jekyll-remote-include.rb [hunk 1]
```diff
@@ -78,7 +78,7 @@ def render(context)
     end
 
     def split_params(params)
-      params.split("|").map(&:strip)
+      params.split("||").map(&:strip)
```
Comment (ianjevans) [inline lib/jekyll-remote-include.rb:81]: Can you update the README with the new token?

Comment (nickvigilante) [reply lib/jekyll-remote-include.rb:81]: Done.

File: README.md [hunk 2]
```diff
@@ -39,10 +39,17 @@ To include the entire file:
...
 
 ```liquid
-{% remote_include https://raw.githubusercontent.com/jekyll/jekyll/master/README.markdown|begin_token|end_token %}
+{% remote_include https://raw.githubusercontent.com/jekyll/jekyll/master/README.markdown||begin_token||end_token %}
+```
+
+URLs can also support using liquid: 
+
+```liquid
+{% assign foo = "https://raw.githubusercontent.com/jekyll/jekyll/master/README.markdown" %}
```
Comment (ianjevans) [inline README.md]:
```suggestion
{% assign x = "https://raw.githubusercontent.com/jekyll/jekyll/master/README.markdown" %}
```

**Review by ianjevans — APPROVED** [review]

**Pull request merged**